.gitignore

```diff
@@ -5,3 +5,12 @@ target
 .classpath
 .java-version
 *.iml
+.DS_Store
```

Comment:
You only need this line, not the rest of it

Reply:
Done